src/main/java/org/tikv/common/log/SlowLogImpl.java

```diff
@@ -72,17 +78,19 @@ public void setError(Throwable err) {
   @Override
   public void log() {
     long currentMS = System.currentTimeMillis();
```

Comment:
should it be removed?

Reply:
we will lose the time point information if removed.
` System.nanoTime()` can be only used to measure the difference of time.

Reply:
fixed